compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/interop/UIKitView.uikit.kt

```diff
@@ -112,6 +142,7 @@ fun <T : UIView> UIKitView(
     onRelease: (T) -> Unit = STUB_CALLBACK_WITH_RECEIVER,
     onResize: (view: T, rect: CValue<CGRect>) -> Unit = DefaultViewResize,
     interactive: Boolean = true,
+    accessibilityEnabled: Boolean = true,
```

Comment:
@igordmn Do you know if it's a compatible change for K/N?

Reply:
I don't know, but we don't guarantee compatibility until iOS Release.

Also, we need to enable binary validator for native before that.